care/emr/management/commands/load_observation_definition.py

```
@@ -0,0 +1,409 @@
+        if options["google_sheet"]:
+            return read_csv_from_google_sheet(
+                options["google_sheet"], options["sheet_name"]
+            )
+        if options["source"]:
+            if options["source"].startswith("http"):
+                return read_csv_from_url(options["source"])
+            return read_csv_from_file(options["source"])
+        raise ValueError("Must provide either source file/URL or --google-sheet")
+
+    def process_row(
+        self, row: dict, facility: Facility | None, validate_codes: bool
```

Ruff (0.14.10)

Unused method argument: `facility`

(ARG002)